### ISSUE TITLE
Swagger 0.6.0 reports invalid version

### ISSUE THREAD

**Issue by themalkolm** [go-swagger/go-swagger#650]
```
$ https://github.com/go-swagger/go-swagger/releases/download/0.6.0/swagger_linux_amd64
$ chmod +x ./swagger_linux_adm64
$ ./swagger_linux_amd64 version
0.5.0-171-gae8082b
```

**Comment by kyteague** [comment]
Also getting this, even from master.  This is really confusing.

    $ go get -u github.com/go-swagger/go-swagger/cmd/swagger
    $ swagger version
    0.5.0-171-g34f62b5

**Comment by casualjim** [comment]
I will make a 0.6.1 release with a new feature, and that should also fix the version then

**Comment by casualjim** [comment]
this is fixed in 0.7.x
